src/test/java/be/syntra/devshop/DevshopFront/testutils/PaymentUtils.java

```diff
@@ -11,7 +11,7 @@
     public static PaymentDto createPaymentDtoWithTotalCartPrice() {
         return PaymentDto.builder()
                 .paymentOptions(Arrays.asList(PaymentOption.values()))
-                .totalCartPrice(new BigDecimal("88.99"))
+                .totalCartPrice(new BigDecimal("88"))
```

Comment:
now I'm quite curious 😄 what test failed because of the decimal numbers?

Reply:
hehe, yeah, a lot of the time i catch myself being in the fixing tests to make them pass mindset.....danger DANGER